### ISSUE TITLE
Uninstalling MathPlayer causes AccessibleObjectFromEvent to fail

### ISSUE THREAD

**Issue by dkager** [nvaccess/nvda#6177]
Based on my report to the nvda-devel mailing list.

This has been consistently observed on Windows 7 and 10, both 64-bit, with NVDA 2016.2.1 and MathPlayer v4.

STR:
-   Remove or rename "C:\Program Files (x86)\Design Science\MathPlayer\MathPlayer.dll".
-   Use NVDA, no restart is required.
-   Restore the DLL. The problems go away.

Examples of things where NVDA speaks and brailles nothing after doing the above:
-   Context menus for items on the desktop and elsewhere in Windows Exlorer.
-   The edit box for renaming a file or folder in Windows Explorer.
-   The treeview in the registry editor.
-   The address bar in IE11.

The log has the following to say about this:

```
IO - inputCore.InputManager.executeGesture (12:38:21):
Input: kb(laptop):applications
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (12:38:21):
oleacc.AccessibleObjectFromEvent with window 722786, objectID -4 and childID 0: [Error -2147312566] Error loading type library/DLL
IO - inputCore.InputManager.executeGesture (12:38:22):
Input: kb(laptop):escape
```

I have seen the same problem after uninstalling SuperNova, but I don't know which specific file caused that problem. There may not be much NVDA can do about this problem since the Win32 API throws an error, but IMO this should be a warning, not a debugWarning.

**Comment by josephsl** [comment]
@nishimotz, do you happen to have the link to your workaround? Thanks.

**Comment by josephsl** [comment]
Hi,

Or it might have been someone else who said telling Windows to register certain DLL's fixes the issue. Thanks.

**Comment by dkager** [comment]
Yes, it looks like both MathPlayer and SuperNova leave crust in the registry to confuse the COM system.

**Comment by nishimotz** [comment]
How to use the Regsvr32 tool and troubleshoot Regsvr32 error messages
https://support.microsoft.com/en-us/kb/249873

I have reported regarding "oleacc.dll" as follows:
https://github.com/nvaccess/nvda/issues/4408#issuecomment-155326311

**Comment by jcsteh** [comment]
The more general issue is #2807, where a possible tool to fix such issues is also being discussed. Marking as a duplicate of #2807.
